networking.go

```diff
@@ -191,3 +195,81 @@ func startSSFUnix(s *Server, addr *net.UnixAddr) <-chan struct{} {
 	}()
 	return done
 }
+
+type SoftnetData struct {
+	Processors []SoftnetDataProcessor
+}
+
+type SoftnetDataProcessor struct {
+	Processed      int64
+	Dropped        int64
+	TimeSqueeze    int64
+	CPUCollision   int64
+	ReceivedRPS    int64
+	FlowLimitCount int64
+}
+
+// SoftnetStat provides information about UDP traffic, as provided
+// on Linux machines by /proc/net/softnet_stat. Its behavior on other
+// operating systems is undefined.
+func SoftnetStat() (SoftnetData, error) {
```

Comment:
This likely should check `runtime` for Linux and squawk? (Although the caller could do that too.